tests/Doctrine/Tests/ODM/PHPCR/Functional/ReorderTest.php

```diff
@@ -102,7 +104,7 @@ public function testReorderNoop()
 
     public function testReorderNoObject()
     {
-        $this->setExpectedException('\Doctrine\ODM\PHPCR\Exception\InvalidArgumentException');
+        $this->expectException(InvalidArgumentException::class);
         $this->dm->reorder('parent', 'first', 'second', false);
         $this->dm->flush();
```

Comment:
I guess this `flush()` isn't needed here, but it's okay.

Reply:
yeah i think you are right. there was a ton of those, where the test probably just was a copy of the happy path with an `@expectedException` annotation at the top, so it was hard to guess where it was expected to explode...